### PR TITLE
update sdrplay

### PR DESCRIPTION
**Pull request by necrose99**
https://github.com/pentoo/pentoo-overlay/tree/641795ee32da0b70466d44e22922315957186140 

some reson repoman is hatting on me.. i;ve tried to manifest in vm... 
bugged in gentoo , however added URLS to arm/arm64 files.

### CODE REVIEW

**Comment by blshkv** [comment]
Please split into two different pulls

**Comment by ZeroChaos-** [comment]
Why does this have an x86 keyword and no x86 download? NACK

File: net-analyzer/sdrplay/sdrplay-2.13.1-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+MY_PV_12=$(get_version_component_range 1-2)
+
+DESCRIPTION="SDRplay API/HW driver for all RSPs"
+HOMEPAGE="https://www.sdrplay.com"
+
+SRC_URI="
+	amd64? ( http://www.sdrplay.com/software/SDRplay_RSP_API-Linux-${PV}.run )
+	arm?   ( https://www.sdrplay.com/software/SDRplay_RSP_API-ARM32-${PV}.run  )
+	arm64? ( https://www.sdrplay.com/software/SDRplay_RSP_API-ARM64-${PV}.run )
+"
```
Comment (dwfreed) [inline net-analyzer/sdrplay/sdrplay-2.13.1-r1.ebuild:20]: This won't download anything for x86, which is in KEYWORDS below

Comment (necrose99) [reply net-analyzer/sdrplay/sdrplay-2.13.1-r1.ebuild:20]: https://gitweb.gentoo.org/repo/gentoo.git/tree/net-wireless/sdrplay/sdrplay-2.13.1-r1.ebuild
copy paste , append... so flipping sleepy i'll drop keyword. 

 @ZeroChaos- @dwfreed  with me buiilding pentoo on arm64 like a machine till 3am over the last few weeks 
sleep has been at well a premium.

**Comment by blshkv** [comment]
Im not big of forking and unforking but ok since you create a pull.  Let's hope Gentoo will fix it soon

**Comment by necrose99** [comment]
bug has be up for year+

**Pull request merged**